applications/StructuralMechanicsApplication/tests/response_function_tests/mass_response_parameters.json

```diff
@@ -10,17 +10,17 @@
         "echo_level"      : 0
     },
     "kratos_response_settings":{
-        "response_type"          : "mass",
+        "response_type"            : "mass",
```

Comment:
In a test like this you should specify the least amount of Parameters to test the defaults

Reply:
now without validation this is the minimal set of settings